2023-risks/m4-inadequate-privacy-controls.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: col-sidebar
+title: "M4: Inadequate Privacy Controls"
+---
+
+# Threat Agents
+
+**Application Specific**
+
+Privacy controls are concerned with protecting Personally Identifiable Information (PII). This information is valuable to attackers for various reasons, e.g., for
+
+- impersonating the victims in cases of fraud,
+- misusing the victims' payment data, or
+- blackmailing the victims with sensitive information.
```

Comment:
- impersonating the victims in cases of fraud,
- misusing the victim's payment data, or
- blackmailing the victim's with sensitive information.
- Absence of data deletion mechanisms
- Limited user awareness and control
- Absence of granular privacy settings